<ui version="4.0" >
 <class>IgSettingsEditorMainWindow</class>
 <widget class="QMainWindow" name="IgSettingsEditorMainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Application Settings Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpen_Settings" />
    <addaction name="actionOpen_INI_File" />
    <addaction name="actionOpen_Mac_Property_List" />
    <addaction name="actionOpen_Windows_Registry_Path" />
    <addaction name="actionRefresh" />
    <addaction name="separator" />
    <addaction name="actionClose" />
   </widget>
   <widget class="QMenu" name="menuOptions" >
    <property name="title" >
     <string>Options</string>
    </property>
    <addaction name="actionAuto_Refresh" />
    <addaction name="actionFallback" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuOptions" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionClose" >
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
  <action name="actionOpen_Settings" >
   <property name="text" >
    <string>Open Application Settings...</string>
   </property>
  </action>
  <action name="actionOpen_INI_File" >
   <property name="text" >
    <string>Open INI File...</string>
   </property>
  </action>
  <action name="actionOpen_Mac_Property_List" >
   <property name="text" >
    <string>Open Mac Property List...</string>
   </property>
  </action>
  <action name="actionOpen_Windows_Registry_Path" >
   <property name="text" >
    <string>Open Windows Registry Path...</string>
   </property>
  </action>
  <action name="actionRefresh" >
   <property name="text" >
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionAuto_Refresh" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Auto-Refresh</string>
   </property>
  </action>
  <action name="actionFallback" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Fallback</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen_Settings</sender>
   <signal>triggered()</signal>
   <receiver>IgSettingsEditorMainWindow</receiver>
   <slot>openSettings()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>253</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_INI_File</sender>
   <signal>triggered()</signal>
   <receiver>IgSettingsEditorMainWindow</receiver>
   <slot>openIniFile()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>253</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Mac_Property_List</sender>
   <signal>triggered()</signal>
   <receiver>IgSettingsEditorMainWindow</receiver>
   <slot>openPropertyList()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>253</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Windows_Registry_Path</sender>
   <signal>triggered()</signal>
   <receiver>IgSettingsEditorMainWindow</receiver>
   <slot>openRegistryPath()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>253</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>IgSettingsEditorMainWindow</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>253</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>IgSettingsEditorMainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>253</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openSettings()</slot>
  <slot>openIniFile()</slot>
  <slot>openPropertyList()</slot>
  <slot>openRegistryPath()</slot>
  <slot>refresh()</slot>
  <slot>setAutoRefresh(bool)</slot>
  <slot>setFallbacksEnable(bool)</slot>
  <slot>about()</slot>
 </slots>
</ui>
